OsmAnd/res/values/strings.xml

```diff
@@ -91,7 +92,7 @@
     <string name="verify_account">Verify your account</string>
     <string name="slope_legend_description">Slope uses colors to visualize terrain steepness. You can read more about Slope in %1$s.</string>
     <string name="terrain_category_description">Terrain contour lines, full hillshade map with dark shades and slope maps with colors to show peaks and lowlands. </string>
-    <string name="terrain_map">Terrain map</string>
+    <string name="terrain_map">3D Relief</string>
```

Comment:
looks very confusing shouldn't be done UI should use relief_3d already trasnlated line

Reply:
Yes, depends on what we finally decide to call that respective download file type. If "3D Relief" (instead of "Terrain map"), we would of course use the existing string "relief_3d". I can adjust the PR as you like. (This concerns only our naming of choice for  DownloadActivityType GEOTIFF_FILE.)